music_assistant/providers/radioparadise/__init__.py

```diff
@@ -238,6 +229,22 @@ def _parse_radio(self, channel_id: str) -> Radio:
             },
         )
 
+        # Add static station icon
+        station_icon = channel_info.get("station_icon")
+        if station_icon:
+            icon_url = f"{STATION_ICONS_BASE_URL}/{station_icon}"
+            images = [
+                MediaItemImage(
+                    provider=self.lookup_key,
+                    type=ImageType.THUMB,
+                    path=icon_url,
+                    remotely_accessible=True,
+                )
+            ]
+            radio.metadata.images = UniqueList(images)
```

Comment:
```suggestion
            radio.metadata.add_image(
                MediaItemImage(
                    provider=self.lookup_key,
                    type=ImageType.THUMB,
                    path=icon_url,
                    remotely_accessible=True,
                )
            )
```

Reply:
Ok now I see. I saw that MediaItemMetadata had an images field and assumed direct assignment was the way to handle it. I should have looked more carefully at the class methods to see if there were purpose-built functions. Thanks for the solution.